app/views/timed_service_price_policies/_adjustment_row.html.haml

```diff
@@ -0,0 +1 @@
+= render "instrument_price_policies/adjustment_row", price_group: price_group, price_policy: price_policy, cancellation: false, minimum_cost: false
```

Comment:
If we do move stuff out of `instrument_price_policies`, does it make sense for the default to now be `false` and you pass in `true` if you want these extra features?